docs/conf/portland/2019/cfp.rst

```diff
@@ -24,46 +23,78 @@ Between the talks on stage, the discussions in the unconference, the collaborati
+The focus of Write the Docs is software documentation, but we actively seek talks that address a wide range of related subjects, at various levels of expertise.
+Documentation perspectives from other fields are welcome, as are topics from adjacent fields!
 
 **Practicality and positivity**
 
-We prefer talks backed by experience and experimentation to talks about theory, and we definitely don't like talks that bad-mouth technologies or approaches. Don't tell us why you hate something – tell us how you overcame the problems it was causing.
+We prefer talks backed by experience and experimentation to talks about theory, and we definitely don't like talks that bad-mouth technologies or approaches.
+Don't tell us why you hate something – tell us how you overcame the problems it was causing.
 
 **Process over tooling**
 
-We tend to avoid talks about specific tools, which often turn into marketing pitches or tutorials. We would much rather hear about process, culture, data, people, or the metaphysical side effects of spending your life thinking about docs.
+We tend to avoid talks about specific tools, which often turn into marketing pitches or tutorials.
+We would much rather hear about process, culture, data, people, or the metaphysical side effects of spending your life thinking about docs.
 
 **Audience awareness**
 
-When crafting talk proposals, remember that you're going to be talking to a mix of levels of expertise, skill sets, and professions. Your talk doesn't have to be relevant to everyone, but it should be relevant to most people and shouldn't
-make too many assumptions about what people already know.
+When crafting talk proposals, remember that you're going to be talking to a mix of levels of expertise, skill sets, and professions.
+Your talk doesn't have to be relevant to everyone, but it should be relevant to most people and shouldn't make too many assumptions about what people already know.
+If you are making those assumptions about what your audience knows, it helps everyone if you state them up front explicitly.
 
-Take a look at the abstracts for accepted talks from the last couple of conferences for some ideas:
+It can be  helpful to check out topics that might be related to your talk from previous years as well:
 
+* `Portland 2018 <http://www.writethedocs.org/conf/portland/2018/speakers/>`_
+* `Prague 2018 <http://www.writethedocs.org/conf/prague/2018/speakers/>`_
 * `Portland 2017 <http://www.writethedocs.org/conf/na/2017/speakers/>`_
 * `Prague 2017 <http://www.writethedocs.org/conf/eu/2017/speakers/>`_
-* `Portland 2016 <http://www.writethedocs.org/conf/na/2016/speakers/>`_
-* `Prague 2016 <http://www.writethedocs.org/conf/eu/2016/speakers/>`_
 
-Need help?
------------
 
-If you need a hand preparing your talk proposal, there are lots of good places to start:
+Who we're looking for
```

Comment:
Oh there it is... I'd move this section to the top where I made the list suggestion, and restructure so that the TL;DR list of roles is at the top, so people who are wary will get the "you belong here <3" message early